docs/index.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: d'Intérieur
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/AY2122S2-CS2103-T12-2/tp/branch/master/graph/badge.svg?token=LTPAH44EQN)](https://codecov.io/gh/AY2122S2-CS2103-T12-2/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**d'Intérieur is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using d'Intérieur, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested about developing d'Intérieur, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```

Comment:
"... interested about developing..." -> "...interested in developing..."

Reply:
Thanks for the feedback, you could try to use the `suggestion` feature for the comment so that I can merge suggestion more quickly rather than having to create a new commit on sourcetree!

Reply:
Okay thanks